pkg/webhook/nodenetworkconfigurationpolicy/conditions.go

```diff
@@ -0,0 +1,107 @@
+package nodenetworkconfigurationpolicy
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+
+	corev1 "k8s.io/api/core/v1"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+const (
+	TimestampLabelKey = "nmstate.io/webhook-mutating-timestamp"
```

Comment:
Could you move this to a separate module and under a separate hook? If we introduce a new hook (other than those for conditions.go), this label would be universal.

Reply:
Well maybe we want two timestamps, one for deleteConditions and another for setConditionsUnknown.